src/User.cpp

```diff
@@ -526,7 +526,7 @@ CIRCNetwork* CUser::AddNetwork(const CString& sNetwork, CString& sErrorRet) {
 }
 
 bool CUser::AddNetwork(CIRCNetwork* pNetwork) {
-    if (FindNetwork(pNetwork->GetName())) {
+    if (FindNetwork(pNetwork->GetName(), CaseSensitivity::CaseSensitive)) {
```

Comment:
If you do it like this, there will be two networks with almost the same name, the difference only in case of letters, no?

Reply:
And then a future `FindNetwork()` will return arbitrarily one or the other

Reply:
Hmm, that is a good point. I initially did it this way to try to restrict the change size, but that is a nasty side effect. 🤔